Sources/SymDiff/source/Transform.cs

```diff
@@ -481,14 +481,29 @@ private static string sanitize(string s)
     { 
       return s.Replace("$", "");
     }
+
+    private static (string, string) RemoveCommonPrefix(string s1, string s2)
+    {
+      int i = 0;
+      while (i < Math.Min(s1.Length, s2.Length) && s1[i] == s2[i]) {
+        ++i;
+      }
+      return (s1.Substring(i), s2.Substring(i));
+
+    }
+
     public static string mkEqProcName(string p1, string p2)
     {
-      var p1WithoutPrefix = p1.Split(".").Last();
-      var p2WithoutPrefix = p2.Split(".").Last();
-      if (p1WithoutPrefix.Equals(p2WithoutPrefix))
+      var p1Parts = p1.Split(".");
+      var p2Parts = p2.Split(".");
```

Comment:
i think the parts are not used anywhere, so can be removed

Reply:
Yeah, that was unintentional. It occurred to me while I was AFK that this drops the (different) prefixes, which I think it's better to keep.